### PR TITLE
Add metrics to track appservice transactions

### PR DESCRIPTION
**Pull request by Half-Shot**
This just adds some simple tracking from synapse to see how many transactions have been sent from the homeserver to an AS in prometheus.

### CODE REVIEW

File: synapse/appservice/api.py [hunk 1]
```diff
@@ -219,12 +238,15 @@ def push_bulk(self, service, events, txn_id=None):
                 args={
                     "access_token": service.hs_token
                 })
+            sent_transactions_counter.label(service.id).inc()
```
[the remark this answers is not in the document]
Comment (turt2live) [reply synapse/appservice/api.py]: to be fair, the ID being a random string was a convention started with the js-bridge library. There's still quite a few bridges which have the default ID "telegram" or similar.

**Comment by Half-Shot** [comment]
@richvdh are you okay to review this?

**Comment by richvdh** [comment]
This might be more in @hawkowl's department actually

**Comment by Half-Shot** [comment]
Apologies, hawkowl? 😇

**Comment by hawkowl** [comment]
Looks fine to me!

**Comment by Half-Shot** [comment]
Kewl, I'll merge then :)

**Pull request merged**